scipy/stats/_stats_py.py

```diff
@@ -5013,11 +5013,11 @@ def pointbiserialr(x, y):
         r_{pb} = \frac{\overline{Y_{1}} -
                  \overline{Y_{0}}}{s_{y}}\sqrt{\frac{N_{1} N_{2}}{N (N - 1))}}
```

Comment:
```suggestion
                 \overline{Y_{0}}}{s_{y}}\sqrt{\frac{N_{0} N_{1}}{N (N - 1))}}
```

Reply:
Drive-by comment: if we're improving this formula, would it be possible to make it easier for those who still do `object?` in ipython --- compare `\overline{Y_0}{s_y} \sqrt{ \frac{N_0 N_1}{N (N-1)}}` or even `\overline{Y_0}{s_y} \sqrt{ N_0 N_1 / N (N-1) }`